src/theia/sfm/feature_extractor_and_matcher.h

```diff
@@ -92,9 +92,10 @@ class FeatureExtractorAndMatcher {
   bool AddImage(const std::string& image_filepath,
                 const CameraIntrinsicsPrior& intrinsics);
 
-  // Sort and fill the vector of masks in order to make a correspondance between
-  // the idexes of 'image_filepaths_' and 'mask_filepaths_'.
-  bool SetMasksForFeaturesExtraction(std::vector<std::string> mask_filepaths);
+  // Add a mask to the 'image_masks_' map.
```

Comment:
I suggest reading comments to correct English mistakes.
(I'm not even sure that this sentence is right...)